docs/api.md

```diff
@@ -99,100 +98,104 @@ nos.claimGas()
     .catch((err) => alert(`Error: ${err.message}`));
 ```
 
-## `testInvoke`
+### `testInvoke`
 The `testInvoke` function executes a test invocation transaction on behalf of the currently
 authenticated account.  It does not require the user to grant permission.
 
-### Parameters
-1. `string` - The script hash of the Smart Contract you want to invoke.
-2. `string` - The operation of the Smart Contract you want to invoke.
-3. `...any` - Zero or more args of the Smart Contract you want to invoke.
+#### Parameters
+1. `scriptHash` [string] - The script hash of the Smart Contract you want to invoke.
+2. `operation` [string] - The operation of the Smart Contract you want to invoke.
+3. `args` [string] - An arguments array of the Smart Contract you want to invoke.
```

Comment:
I think we should change `[string]` to `[string[]]` or `[array<string>]`.

Reply:
Yes, good call.